apps/nextjs/src/middlewares/csp.ts

```diff
@@ -2,31 +2,27 @@ import { NextRequest, NextResponse } from "next/server";
 import { v4 as uuidv4 } from "uuid";
 
 function generateNonce(): string {
-  if (typeof crypto !== "undefined" && crypto.getRandomValues) {
-    const array = new Uint8Array(16);
-    crypto.getRandomValues(array);
-    return btoa(String.fromCharCode.apply(null, array as unknown as number[]));
-  } else {
-    // Use uuid library to generate a random value
-    return uuidv4();
-  }
+  return uuidv4();
```

Comment:
The uuid package says "Cryptographically-strong random values", but it would still sit slightly better with me if there was a way to keep using crypto in production. This feels like a bit of a downgrade, but I don't think it would matter in practicality

Reply:
I've since discovered "node:crypto" so perhaps we merge this and then investigate that?